cmd/kubeadm/app/util/system/package_validator_test.go

```diff
@@ -224,8 +228,14 @@ func TestValidatePackageVersion(t *testing.T) {
 			if test.err != nil {
 				if err == nil {
```

Comment:
This is better to be `len(err) == 0`, furthermore, `err` needs to be called `errs` here as it's a slice now.

Reply:
you are correct. I will update.